packages/bpk-mixins/src/mixins/_forms.scss

```diff
@@ -666,7 +670,8 @@
 
 @mixin bpk-radio--invalid {
   input[type='radio'] {
-    background: url(map-get($bpk-elements, radio-invalid)) no-repeat;
+    border: ($bpk-one-pixel-rem * 3) solid $bpk-color-panjin;
```

Comment:
Is it possible to use the [border size tokens](https://backpack.github.io/tokens/borders) for this? Just to simplify things.

Reply:
It might be that we don't wanna use border size because it's not _exactly_ a border, it's more intrinsic to the component's style. Either way, `($bpk-one-pixel-rem * 3) solid <color>` is repeated in this file a couple of times, so we might wanna declare a mixin or variable for this so it's DRYer

Reply:
Sure thing, forgot we actually have a 3px border token!